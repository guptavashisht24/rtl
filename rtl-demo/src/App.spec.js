import React from 'react';
import { render, cleanup } from '@testing-library/react';
import "jest-dom/extend-expect";
import App from './App';

afterEach(cleanup);

it('matches snapshots', ()=>{
  const { asFragment } = render(<App />)
  expect(asFragment()).toMatchSnapshot();
})
