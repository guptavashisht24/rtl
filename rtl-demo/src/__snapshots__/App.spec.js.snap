// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshots 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <header
      class="App-header"
    >
      <img
        alt="logo"
        class="App-logo"
        src="logo.svg"
      />
      <p>
        Edits and checks 
        <code>
          src/App.js
        </code>
         and save to reload.
      </p>
      <a
        class="App-link"
        href="https://reactjs.org"
        rel="noopener noreferrer"
        target="_blank"
      >
        Learn React
      </a>
    </header>
  </div>
</DocumentFragment>
`;
